src/auth/auth.helper.ts

```diff
@@ -20,7 +20,9 @@ export class AuthHelper {
 	}
 
 	public async validateUser(decoded: any): Promise<User> {
-  		return this.userRepository.findOne(decoded.id);
+  		return this.userRepository.findOne({
+			where: { id: decoded.id } }
+		);
```

Comment:
is indentation ok?
or Github has an issue

Reply:
da, arata ok <img width="602" alt="image" src="https://user-images.githubusercontent.com/57131600/185393512-21e5408b-ea95-4325-8e1f-71afb25498e0.png">